testutils/src/main/java/com/amplifyframework/testutils/sync/SynchronousAuth.java

```diff
@@ -392,4 +393,19 @@ public void signOut() throws AuthException {
                 )
         );
     }
+
+    /**
+     * Sign out synchronously with options.
+     * @param options Advanced options for signing out
+     * @throws AuthException exception
+     */
+    public void signOut(AuthSignOutOptions options) throws AuthException {
+        Await.<Object, AuthException>result(AUTH_OPERATION_TIMEOUT_MS, (onResult, onError) ->
+                asyncDelegate.signOut(
+                    options,
+                    () -> onResult.accept(new Object()),
```

Comment:
Richard added a `VoidResult` sentinel object that could be used here

Reply:
```suggestion
                    () -> onResult.accept(VoidResult.instance()),
```